XCUITests/FxScreenGraph.swift

```diff
@@ -596,6 +599,18 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> Scree
         }
     }
 
+    // This menu is only available for iPhone, NOT for iPad, no menu when long tapping on tabs button
+    if !isTablet {
+        map.addScreenState(TabTrayLongPressMenu) { screenState in
+            screenState.dismissOnUse = true
+            screenState.tap(app.buttons["New Tab"], to: NewTabScreen)
+            screenState.tap(app.buttons["New Private Tab"], forAction: Action.OpenPrivateTabLongPressTabsButton, transitionTo: NewTabScreen) { userState in
+                userState.isPrivate = !userState.isPrivate
+            }
+            screenState.tap(app.buttons["Close Tab"], to: HomePanelsScreen)
```

Comment:
This needs to be an action here.

```swift
screenState.tap(app.tables["Context Menu"].cells["action_remove"], forAction: Action.CloseTabFromTabTrayLongPressMenu, Action.CloseTab, tansitionTo: HomePanelsScreen)
```

Reply:
Yup, that was my first attempt, but was not sure because the app state does not change, so I did that way without action. I will change it as thought initially